### PR TITLE
Feature/add covers to tests

### PR DESCRIPTION
**Pull request by levelfivehub**
@Maks3w 

Added covers annotation to unit tests.  Unit test coverage is coming up lower.

### CODE REVIEW

File: test/Aggregate/AggregateHydratorFunctionalTest.php [hunk 1]
```diff
@@ -22,6 +22,7 @@
 
 /**
  * Integration tests {@see AggregateHydrator}
+ * @covers Zend\Hydrator\Aggregate\AggregateHydrator<extended>
```
Comment (Maks3w) [inline test/Aggregate/AggregateHydratorFunctionalTest.php]: Functional tests are not unit test. Revert.

File: test/HydratorClosureStrategyTest.php [hunk 2]
```diff
@@ -12,6 +12,10 @@
 use Zend\Hydrator\ObjectProperty;
 use Zend\Hydrator\Strategy\ClosureStrategy;
 
+/**
+ * @covers Zend\Hydrator\ObjectProperty<extended>
```
Comment (Maks3w) [inline test/HydratorClosureStrategyTest.php]: Revert this test not match the 1:1 path.

File: test/HydratorObjectPropertyTest.php [hunk 3]
```diff
@@ -11,6 +11,9 @@
 
 use Zend\Hydrator\ObjectProperty;
 
+/**
```
Comment (Maks3w) [inline test/HydratorObjectPropertyTest.php]: Revert this test not match the 1:1 path.

File: test/HydratorStrategyTest.php [hunk 4]
```diff
@@ -12,6 +12,9 @@
 use Zend\Hydrator\HydratorInterface;
 use Zend\Hydrator\ClassMethods;
 
+/**
```
Comment (Maks3w) [inline test/HydratorStrategyTest.php]: Revert this test not match the 1:1 path.

File: test/HydratorTest.php [hunk 5]
```diff
@@ -29,6 +29,15 @@
 use ZendTest\Hydrator\TestAsset\ObjectProperty as ObjectPropertyAsset;
 use ZendTest\Hydrator\TestAsset\ArraySerializable as ArraySerializableAsset;
 
+/**
```
Comment (Maks3w) [inline test/HydratorTest.php]: Too much covers. Maximum 1. Remember covers annotations only in unit (1) test

Comment (levelfivehub) [inline test/HydratorTest.php]: Now just the one.

**Comment by levelfivehub** [comment]
Reverted.

**Comment by weierophinney** [comment]
Thanks, @levelfivehub

**Pull request merged**